test/Unit.test.js

```diff
@@ -6,8 +6,11 @@ import unit from '../src/Unit'
 
 // TODO: Implement implicit conversion of strings to units in add, sub, mul, etc.
 
-describe('unitmath', () => {
+// TODO: Remove once uses of `math`, and `Unit` are removed or these values
+//       are imported.
+/* global math, Unit */
```

Comment:
@harrysarson, I've never seen this before. What does it do?

Reply:
Aha, tells the linter to ignore those undefined variables that are still present in the skipped tests?

Reply:
Exactly, these can and should be removed soon.